go/bundle/README.md

```diff
@@ -37,11 +45,33 @@ Once you have the har file, generate the bundled exchange file via:
 gen-bundle -har foo.har -o foo.wbn
 ```
 
+#### From a URL list
+
+`gen-bundle` also accepts `-URLList FILE` flag. `FILE` is a plain text file with one URL on each line. `gen-bundle` fetches these URLs and put the responses into the bundle. For example, you could create `urls.txt` with:
+
+```
+# This is a comment.
```

Comment:
nit:

# A line starting with '#' is a comment.

Reply:
done